.github/workflows/conformance.yml

```diff
@@ -21,7 +23,7 @@ concurrency:
   cancel-in-progress: true
 
 permissions:
-  contents: read
+  contents: write # needed for uploading release artifacts
```

Comment:
we should add the permissions at the job level and leave this to `read` or OpenSSF will complain

Reply:
Ok, I can do that. I figured it wouldn't matter since there's only one job, but I guess it makes sense if we add more.